### ISSUE TITLE
Consider Updating Vendoring Documentation

### ISSUE THREAD

**Issue by kirikintha** [mozilla-neutrino/neutrino-dev#861]
### Bug or issue?

Documentation

Please try to answer the following questions:

- What version of Neutrino are you using?
V8

### Feature request or enhancement?

Would you consider a small documentation change on: https://neutrino.js.org/packages/react#vendoring

To include a reference to https://neutrino.js.org/packages/react/#vendoring when trying to use the vendor prefix in react projects.

These lines:

> External dependencies are automatically split into separate chunks from the application code, by the new webpack SplitChunksPlugin.

could be considered ambiguous to developers who wish to gain the vendor.[chunk-hash] bundle for their applications.

For example: 

> However, if you do need to use the vendor entry options or have used them in the past, please refer to... https://neutrino.js.org/packages/react/#vendoring for more information.

May help users who have the react plugin, or are used to having a separate vendor file and are migrating to neutrino. I have been migrating to V8 from V5, and was confused by the references and was not entirely sure which option suited my build.

Thanks a ton for all your hard work!

**Comment by edmorley** [comment]
Hi! Thank you for filing about improving the docs - it's often hard to get feedback about them :-)

Last week the way we hosted the docs was changed, but unfortunately it resulted in the `master` version of the docs was being shown on https://neutrino.js.org rather than the docs for Neutrino v8. From the description above it sounds like you might be seeing a cached version of the `master` docs, or else are reading the docs from the repository? (Neutrino v8 is using webpack 3 which doesn't support SplitChunksPlugin -- we haven't yet released Neutrino v9 which will come with webpack 4.)
